runtime/libffi/z/ffi.c

```diff
@@ -267,33 +267,33 @@ ffi_prep_args (unsigned char *stack, extended_cif *ecif)
 	    *(unsigned char *) arg_ptr = * (unsigned char *) (* p_argv);
 					arg_ptr += 3;
 	    break;
- 
+
 	  case FFI_TYPE_SINT8:
 	    *(signed char *) arg_ptr = * (signed char*) (* p_argv);
 			arg_ptr += 3;
 	    break;
- 
+
 	  default:
 	    FFI_ASSERT (0);
 	    break;
         }
       arg_ptr += size;
     }
-  
+
 }
 
 /*======================== End of Routine ============================*/
- 
+
 /*====================================================================*/
 /*                                                                    */
-/* Name     - ffi_prep_cif_machdep.                                   */
+/* Name     - ffi_prep_cif_machdep_core.                              */
 /*                                                                    */
-/* Function - Perform machine dependent CIF processing.               */
+/* Function - Core implementation of machine dependent CIF processing.*/
 /*                                                                    */
 /*====================================================================*/
- 
+
 ffi_status
-ffi_prep_cif_machdep(ffi_cif *cif)
+ffi_prep_cif_machdep_core(ffi_cif *cif)
```

Comment:
This should be static, like the 64-bit version.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/22396/commits/523147ec00bb79f9113161a74fb9254769aa98f5